lib/index.js

```diff
@@ -192,27 +153,13 @@ export default {
           return [];
         }
 
-        // Construct the list of linters to be passed to the CLI by looking at
-        // the linters made available for our language grammar by the LinterMap,
-        // and whether or not the engines are enabled in a user's config file.
-        if (Object.prototype.hasOwnProperty.call(linterMap, grammarName) === true) {
-          // for (const linter of linterMap[grammarName]) {
-          linterMap[grammarName].forEach((linter) => {
-            linterNames.add(linter);
-          });
-        }
-        const configEnabledEngines = getEnabledEngines(configurationFilePath);
-        const linterEnabledEngines = Array.from(linterNames).filter(
-          linterName => configEnabledEngines.has(linterName));
-
         // Construct the command line invocation which runs the Code Climate CLI
         const relpath = atom.project.relativizePath(filePath).pop();
-        const execArgs = ['analyze', '-f', 'json'].concat(
-          getEngineArgs(linterEnabledEngines),
-          [relpath, '</dev/null']);
+        const execArgs = ['analyze', '-f', 'json', relpath];
         const execOpts = {
           cwd: dirname(configurationFilePath),
           uniqueKey: `linter-codeclimate::${relpath}`,
+          env: process.env,
```

Comment:
When executing `codeclimate analyze -f json -e fixme styles.css` with:

- the current master version of this Atom package (with the `</dev/null` stuff removed)
- and codeclimate v0.63.x
- and a test CSS file

in...

- ... the terminal it works and the command returns the json string:
```
[{"categories":["Bug Risk"],"check_name":"FIXME","description":"FIXME found","location":{"lines":{"begin":20,"end":20},"path":"style.css"},"type":"issue","fingerprint":"b5a5eb86d3be898eebbde98668a6cae5","severity":"minor","engine_name":"fixme"},
{"categories":["Bug Risk"],"check_name":"BUG","description":"BUG found","location":{"lines":{"begin":21,"end":21},"path":"style.css"},"type":"issue","fingerprint":"8e1aa283f29fa773f5cc145033fc0a9c","severity":"minor","engine_name":"fixme"},
{"categories":["Bug Risk"],"check_name":"XXX","description":"XXX found","location":{"lines":{"begin":22,"end":22},"path":"style.css"},"type":"issue","fingerprint":"04678738bf48e8539f9060986d3af186","severity":"minor","engine_name":"fixme"}]
```
- ... atom it fails, I see in the console log the message `linter-codeclimate:: Command: `codeclimate analyze -f json -e fixme style.css` and I get the following error in a notification:
```
Exception details:
- COMMAND: `codeclimate analyze -f json -e fixme style.css`
- MESSAGE: ! analyze
! -f
! json
! -e
! fixme
! style.css
error: (Errno::ENOENT) No such file or directory @ rb_sysopen - /code/.codeclimate.yml
Hide Stack Trace
Error: ! analyze
! -f
! json
! -e
! fixme
! style.css
error: (Errno::ENOENT) No such file or directory @ rb_sysopen - /code/.codeclimate.yml
    at ChildProcess.<anonymous> (/opt/cgalvarez/linter-codeclimate/node_modules/sb-exec/lib/index.js:56:20)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Socket.<anonymous> (internal/child_process.js:334:11)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:493:12)
```

I am **NOT** appending `</dev/null` in any case. This same command works for codeclimate versions prior to v.63.x. That's why I think it is an issue from codeclimate. Maybe it is an issue with the package `sb-exec` used by Helpers, but I'm not into that since I only change the codeclimate version used and it works/fails depending exclusively on that change.